src-built-in/components/myApps/src/components/AppDefinition.jsx

```diff
@@ -43,11 +43,14 @@ export default class AppDefinition extends React.Component {
 		// So lets spawn from URL
 		if (this.props.app.url) {
```

Comment:
If we have the time for some improvements I would do this
```javascript
 const { url } = this.props.app
```
And save few bytes and enhance readability

Reply:
next time :-D